src/functions-and-arrays.js

```diff
@@ -52,7 +148,36 @@ const wordsUnique = [
   'bring'
 ];
 
-function uniquifyArray() {}
+function uniquifyArray(mixedWordsArr) {
+  let mixedWordsArrLength = mixedWordsArr.length;
+  console.log("MixedWordsArrLength ===>", mixedWordsArrLength)
+  let newUniqArr = [];
+
+  if(mixedWordsArrLength === 0) {
+    return null;
+  }
+
+
+
+  mixedWordsArr.forEach( (singleWord)=> {
+    console.log("singleWord ==>", singleWord)
+    for(let i = 0; i < mixedWordsArrLength; i++){
+      if (singleWord !== mixedWordsArr[i]){
```

Comment:
Please check this logic again, it should just check if the element is already in the new array